modules/core/src/main/scala/schema.scala

```diff
@@ -1011,6 +1011,8 @@ object Value {
         value.success
       case (s: ScalarType, Some(value: BooleanValue)) if !s.isBuiltIn =>
         value.success
+      case (s: ScalarType, Some(enumValue: EnumValue)) if !s.isBuiltIn =>
+        StringValue(enumValue.name).success
```

Comment:
no need to handle EnumValue for scalar, always allows to skip the apostrophes
the other option would be
```
        enumValue.success
```
that allows to control whether to accept enum-style (with no apostrophes) values for scalars,
but also requires more handling later, e.g.:
```
object IntervalValue {
  def unapply(s: Value): Option[Duration] = s match {
    case StringValue(value) => Try(Duration.parse(value)).toOption
    case EnumValue(name) => Try(Duration.parse(name)).toOption
    case _ => none
  }
}
```
instead of
```
object IntervalValue {
  def unapply(s: StringValue): Option[Duration] =
    Try(Duration.parse(s.value)).toOption
  }
}
```

Reply:
If this _is_ allowed by the spec, then I think you've made the right choice here.